src/main/resources/application.yml

```
@@ -159,7 +159,7 @@ ide-sidecar:
     # How often to check?
     interval-seconds: 60
   proxy:
-    ccloud-api-control-plane-regex: "(/artifact.*)|(/fcpm/v2/compute-pools.*)"
+    ccloud-api-control-plane-regex: "(/artifact.*)|(/fcpm/v2/compute-pools.*)|(/api/metadata/security/v2alpha1/authorize)"
```

If the base url is `https://api.confluent.cloud` -- then the `/api/metadata/..` probably won't work. Can we check to see if `https://api.confluent.cloud/metadata/security/v2alpha1/authorize` is a valid way to call the endpoint? If so, let's do:
```suggestion
    ccloud-api-control-plane-regex: "(/artifact.*)|(/fcpm/v2/compute-pools.*)|(/metadata/security/v2alpha1/authorize)"
```

---

thank you for the comment! @rohitsanj. I have update that. but currently still fixing some test errors. will let you know!